tensorflow_recommenders_addons/dynamic_embedding/core/utils/utils.h

```diff
@@ -42,4 +42,24 @@ This code is for compatibility.*/
 }  // namespace recommenders_addons
 }  // namespace tensorflow
 
+// For propagating errors when calling a function but not return status.
+#if TF_VERSION_INTEGER >= 2130
```

Comment:
The code is duplicated here, will this work?
```
#define LOG_IF_ERROR(...)                  \
  do {                                     \
    const auto _status = (__VA_ARGS__);    \
    if (TF_PREDICT_FALSE(!_status.ok())) { \
      #if TF_VERSION_INTEGER >= 2130       \
        MAYBE_ADD_SOURCE_LOCATION(_status) \
        LOG(ERROR) << _status.message();   \
      #else                                \
        LOG(ERROR) << _status.error_message(); \
      #endif                               \
    }                                      \
  } while (0)
```

Reply:
if I am not mistaken, the two compatibility define macro for different TF version will follow the the criterion "if TF_VERSION_INTEGER >= 2130" be compiled.